cluster/manifests/skipper/deployment.yaml

```diff
@@ -35,6 +35,9 @@ spec:
         prometheus.io/scrape: "true"
 {{- if eq .ConfigItems.skipper_topology_spread_enabled "true" }}
         zalando.org/topology-spread-timeout: 7m
+{{- end }}
+{{- if eq .ConfigItems.skipper_open_policy_agent_enabled "true" }}
+        iam.amazonaws.com/role: "{{ .ConfigItems.skipper_open_policy_agent_instance_role_arn }}"
```

Comment:
This is sth that I'd like to discuss. This will be used to read bundles from S3. It would be better to use a service account IMO and assume a role bound to Skipper. Any preferences?

Reply:
Yes, we should use service account (the OIDC based model) this is used in all other components in this repo. Ideally you can also provision a dedicated role as part of the cluster so it's all managed in one place. See how we do it for other roles here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/d36fd80bc9a7671a9adb20dae588912baede3a45/cluster/cluster.yaml#L1520-L1654 we also have a role naming pattern including the LocalID, which you can follow as well. The idea is that we in theory can have more than one cluster per account (though we don't do it in practice at the moment).

Reply:
Sounds good, I added a new role following the same pattern and assigned it to the existing `skipper-ingress` service account.